improver_tests/calibration/rainforests_calibration/test_ApplyRainForestsCalibrationLightGBM.py

```diff
@@ -208,7 +208,37 @@ def test__prepare_features_dataframe(ensemble_features):
     with pytest.raises(ValueError):
         ApplyRainForestsCalibrationLightGBM(
             model_config_dict={}
-        )._prepare_features_dataframe(ensemble_features)
+        )._prepare_features_array(ensemble_features)
+
+
+def test__evaluate_probabilities(
+    ensemble_features, dummy_lightgbm_models, error_threshold_cube
+):
+    """Test that _evaluate_probabilities populates error_threshold_cube.data with
+    probability data."""
+    plugin = ApplyRainForestsCalibrationLightGBM(model_config_dict={})
+    plugin.tree_models, plugin.error_thresholds = dummy_lightgbm_models
+    input_dataset, feature_variables = plugin._prepare_features_array(ensemble_features)
+    precipitation_ind = feature_variables.index("lwe_thickness_of_precipitation_amount")
+    precip_forecast = input_dataset[:, precipitation_ind]
```

Comment:
Following on from the comment regarding usage of `forecast_cube` in `_calculate_error_probabilities`, I think we should add `ensemble_forecast` to the test signature and use this when referring to forecast data.

Reply:
Done